cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh

```diff
@@ -52,7 +52,7 @@ else
 fi
 
 # 4. Submit a sample request to endpoint
-data_path="./text_to_image_batch_data/batch_data"
+data_path="./text_to_image_batch_data"
```

Comment:
is this required? Have we verified it?

Reply:
yes As per discussion with Madhu updated it @MadhuM02 please feel free to provide your comments